server/views/pages/adjustments/unused-deductions/days.njk

```diff
@@ -0,0 +1,88 @@
+{% extends "../../../partials/layout.njk" %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% if model.addOrEdit === 'edit' %}
+{% set pageContentTitle = "Edit the number of unused deductions" %}
+{% else %}
+{% set pageContentTitle = "Enter the number of unused deductions" %}
+{% endif %}
+{% set pageTitle = applicationName + " - " + pageContentTitle %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block beforeContent %}
+{{ super() }}
+<nav>
+  {{ govukBackLink({
+    text: "Back",
+    href: "/" + prisoner.prisonerNumber
+  }) }}
+</nav>
+{% endblock %}
+
+{% block content %}
+
+{% if model.form.errors.length %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: model.form.errorList()
+    }) }}
+  </div>
+</div>
+{% endif %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-xl">Adjustments</span>
+      {{ pageContentTitle }}
+    </h1>
+  </div>
+</div>
+<form class="form" method="post">
+  <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+  {{ govukInput({
+    label: {
+      html: "<strong>Number of days</strong>"
+    },
+    classes: "govuk-input--width-5",
+    id: "days",
+    name: "days",
+    inputmode: "numeric",
+    value: model.form.days,
+    suffix: {
+      text: "days"
+    },
+    spellcheck: false
+  }) }}
+  {{ govukInput({
```

Comment:
I did write it in the way you suggested but I thought it wasn't an ideal solution because it would be looking it up on each POST rather than once on the GET. 
I'll revert this back to my original implementation